### DISCUSSION TITLE
failed to send otp

### DISCUSSION THREAD

**Discussion by Kushwaha2021** [saran-surya/email_auth#31] [Q&A]
email-auth >> Failed to send OTP ❌
I/flutter (11094): email-auth >> Message from server :: null

**Comment by saran-surya** [comment]
Hi @Kushwaha2021 , Are you using the test or the custom production server ? Seems from your another question. you must have not set up the production server. Kindly check the other issue for the answer. 

https://github.com/saran-surya/email_auth/discussions/29#discussioncomment-1366960

Thanks

**Comment by Kushwaha2021** [comment]
Testing server is fine but production server is not working properly.

**Reply by saran-surya** [reply]
That is what, if you follow the instructions from here, it should be easy

https://saran-surya.github.io/email-auth-node/
